src/SDL.c

```diff
@@ -251,10 +251,27 @@ void SDL_SetMainReady(void)
 // Initialize all the subsystems that require initialization before threads start
 void SDL_InitMainThread(void)
 {
+    static bool done_info = false;
+
     SDL_InitTLSData();
     SDL_InitEnvironment();
     SDL_InitTicks();
     SDL_InitFilesystem();
+
+    if (!done_info) {
+        const char *value;
+
+        SDL_LogInfo(SDL_LOG_CATEGORY_SYSTEM, "SDL %d.%d.%d, revision: %s",
+                    SDL_MAJOR_VERSION, SDL_MINOR_VERSION, SDL_MICRO_VERSION, SDL_REVISION);
```

Comment:
In practice this should typically log something like:

```
SDL 3.1.3, revision SDL-3.1.3 (Debian 3.1.3+ds-1)
```

or maybe

```
SDL 3.1.2, revision SDL-prerelease-3.1.2-1181-gc5a99f151
```

Suitable?

Reply:
Oh, I didn't realize the revision included the version. We probably just need that.

Reply:
Yes, `showrev.sh` is careful to include the version number (even if no git info is available) so I think we should be OK with just the revision.

Reply:
OK, it'll now look something like

```
SDL revision: SDL-3.1.3 (Debian 3.1.3+ds-1)
```

or

```
SDL revision: SDL-prerelease-3.1.2-1181-gc5a99f151
```

Reply:
Great.

Reply:
Can we add the SDL revision after the app info?